elodie/filesystem.py

```diff
@@ -213,13 +218,38 @@ def process_file(self, _file, destination, media, **kwargs):
             shutil.move(_file, dest_path)
             os.utime(dest_path, (stat.st_atime, stat.st_mtime))
         else:
-            shutil.copy2(_file, dest_path)
+            shutil.copy(_file, dest_path)
+            self.set_date_from_filename(dest_path)
 
         db.add_hash(checksum, dest_path)
         db.update_hash_db()
 
         return dest_path
 
+    def set_date_from_filename(self, file):
```

Comment:
I think this should be a function that sets the created/modified time on a file. The mechanism by which that happens can be contained within the function.

My suggestion in pseudocode

```
function set_date_on_file(file) {
    if (config.file_path_date_format) {
        if (file.match(config.file_path_date_format)) {
            // parse the file path and set the date
            return;
        }
    }
    
    try {
        os.utime(dest_path, (stat.st_atime, stat.st_mtime))
    } catch () {
        log('could not set time on file');
    }
}  
```

This addresses the error thrown on a mounted NFS drive when trying to call `copy2` but adds the ability to supercede that with a custom path.